src/wwwroot/js/genpage/gentab/currentimagehandler.js

```diff
@@ -180,10 +180,10 @@ class ImageFullViewHelper {
         else {
             img.style.imageRendering = '';
         }
-        if (newHeight > 101) {
+        if (newHeight > 101 && this.showMetadata) {
             this.toggleMetadataVisibility(false);
         }
-        else if (newHeight < 101) {
+        else if (newHeight < 101 && !this.showMetadata && zoom < 1) {
```

Comment:
This change is required. May seem weird, but add a `console.log()` to both cases to verify.

We only want to trigger `this.toggleMetadataVisibility()` when necessary - meaning:

* if an image shows the metadata, zooming out should not trigger toggleMetadataVisibility
* if an image does not show the metadata, zooming out should trigger toggleMetadataVisibility when zoom level < 101
* if an image shows the metadata, zooming in should trigger toggleMetadataVisibility when zoom level > 101
* if an image does not show the metadata, zooming in should not trigger toggleMetadataVisibility

Reply:
... Why? Retriggering it has no side effects.